2025/shell_scripting/user_management.sh

```
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+<< info
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)